src/Core/Mesh/DCEL/HalfEdge.hpp

```diff
@@ -30,7 +31,7 @@ class RA_CORE_API HalfEdge : public IndexedObject {
               const HalfEdge_ptr& prev  = nullptr,
               const HalfEdge_ptr& twin  = nullptr,
               const Face_ptr&     f     = nullptr,
-              const Index&        index = Index::INVALID_IDX() ); // Build a complete Halfedge
+              const Index&        index = Index::INVALID_IDX() ); // Build a complete Halfedg
```

Comment:
Introducing a typo in a comment. How dare you !

Reply:
(sometimes I hit keyboard without notice ...)